torch/_dynamo/utils.py

```diff
@@ -1578,9 +1578,47 @@ def to_numpy_helper(___tmp_0):
     def convert(obj):
         if isinstance(obj, torch_np.ndarray):
             return obj.tensor.numpy()
+        elif isinstance(obj, torch.Tensor):
+            return obj.numpy()
         else:
             return obj
 
     if isinstance(___tmp_0, tuple):
         return tuple([convert(obj) for obj in ___tmp_0])
     return convert(___tmp_0)
+
+
+def to_tensor(value):
+    def convert(obj):
+        if isinstance(obj, np.ndarray):
+            return torch.from_numpy(obj)
```

Comment:
This should not be necessary, as we should never have a `np.ndarray` inside the graph.

Reply:
`to_tensor()` is also used in wrapping graph input, which may be a `numpy.ndarray`.